bitrise.yml

```diff
@@ -567,6 +593,28 @@ workflows:
 
   RunUnitTests:
     steps:
+    - script@1.1:
```

Comment:
We need to add this part here, which is similar to the workflow Detect_warnings, but there is not a way (at least that I know) to pass an argument to the workflow or the step so that we could do this without duplication